src/screenControl.py

```diff
@@ -115,14 +122,27 @@ def outputBottom(self):
         borderY = maxy - 2
         # first output text since we might throw an exception during border
         usageStr = {
-            SELECT_MODE: SHORT_NAV_USAGE,
-            X_MODE: SHORT_NAV_USAGE,
+            SELECT_MODE: self.getShortNavUsageString(),
+            X_MODE: self.getShortNavUsageString(),
             COMMAND_MODE: SHORT_COMMAND_USAGE
         }[self.mode]
         borderStr = '_' * (maxx - self.getMinX() - 0)
         self.printer.addstr(borderY, self.getMinX(), borderStr)
         self.printer.addstr(borderY + 1, self.getMinX(), usageStr)
 
+    def getShortNavUsageString(self):
+        navOptions = [SHORT_NAV_OPTION_SELECTION_STR,
+                      SHORT_NAV_OPTION_NAVIGATION_STR,
+                      SHORT_NAV_OPTION_QUICK_SELECT_STR,
+                      SHORT_NAV_OPTION_COMMAND_STR]
+
+        # it does not make sense to give the user the option to "open" the selection
+        # in all-input mode
+        if not self.flags.getAllInput():
+            navOptions.insert(2, SHORT_NAV_OPTION_OPEN_STR)
```

Comment:
ah yeah, removing is a bit cleaner imo. it just felt like happing to map the index with 0-offset back to the original array to figure out where it would be inserted was a bit tough to read. but a super small minor nit of course